polaris-patterns/.storybook/main.js

```diff
@@ -14,6 +14,12 @@ module.exports = {
     '@storybook/addon-toolbars',
     '@storybook/addon-viewport',
   ],
+  babel: (config) => {
+    return {
+      ...config,
+      rootMode: 'upward',
+    };
+  },
```

Comment:
This gets the top monorepo babel config, which I can do now because I don't need the `@shopify/react-i18n/babel` plugin